bin/hsplotter

```diff
@@ -23,24 +34,39 @@ def main():
     parser.add_argument("-o", "--out", help="Path where figures are saved.")
     args = parser.parse_args()
     run_date = pd.Timestamp(args.run).to_pydatetime()
+    map = args.map
```

Comment:
Can you change the variable name map to something else, like map_file? "map" is a Python standard library function, and while it is not being used here, and the program still runs, it is good practice not to overwrite Python standard library function names.

Reply:
I made changes to hsplotter, making sure not to rewrite over "map". I also fixed some tab errors in HREFv2ModelGrid and TrackModeler that I made when changing the scripts.